### ISSUE TITLE
bug: Jan is unable to quit when Quick Ask is activated

### ISSUE THREAD

**Issue by imtuyethan** [janhq/jan#3679]
### Problem Statement

- Quick Ask feature is causing issues by not terminating when Jan is closed

## Scope

- Jan should exit fully (we have outlying bug where Jan does not quit if Quick Ask is activated)

### Out-of-scope

- Any enhancements (get pushed to subsequent issue)
- When Jan is triggered through the shortcut, Jan's window opens and generates the response without needing to open the full program window each time (https://discord.com/channels/1107178041848909847/1284230653512257607/1284230653512257607)
- Allow temporary chat

**Comment by imtuyethan** [comment]
Close & follow up on this epic instead: https://github.com/janhq/jan/issues/3944
